webapp/src/main/java/org/opentestsystem/rdw/admin/service/impl/DefaultCsvValidationService.java

```diff
@@ -46,6 +47,7 @@
     private static final String ValidationFormat = "Row: %1$d Failure: %2$s\n";
     private static final String MoreFailuresFormat = "With %1$d additional validation failures\n";
     private static final int MaxFailureMessages = 100;
+    static final int MaxStudentsPerGroup = 200;
```

Comment:
It could be, but we should either discuss how to ensure the configuration value is shared between the admin and group-processor projects.  I suppose we could always just add comments saying that the two properties must be changed in lock-step.  But I think you're right, that may be a phase 2 issue.  The client may want this UX to completely change in phase 2.